tools/sigh.js

```diff
@@ -212,6 +213,27 @@ async function tsc() {
   return result;
 }
 
+async function tslint(args) {
+  let jsSources = [...findProjectFiles(process.cwd(), fullPath => {
+    if (/intermediate/.test(fullPath)) {
```

Comment:
Isn't this renamed to `ts-build` now? It looks like this is still in the `lint()` method too, maybe needs to be updated after https://github.com/PolymerLabs/arcs/pull/1757

Reply:
Indeed, will fix.